storage-node/src/services/webApi/controllers/filesApi.ts

```diff
@@ -42,38 +47,59 @@ export async function getFile(
   try {
     pinDataObjectIdToCache(dataObjectId)
   } catch (err) {
+    logger.error(`Error pinning file ${dataObjectId} to cache`)
     res.status(404).send()
     return
   }
 
-  const unpin = _.once(() => {
+  const unpin = _.once(async () => {
```

Comment:
```suggestion
  const unpin = _.once(() => {
```
Why are you making this async? The underlying call are not async.